Find a functionally equivalent code:android.media.RemoteController.seekTo:COMMENT
Method Modifier: public      
Comment:/**
 * Sets the new playback position.
 * This method can only be called on a registered RemoteController.
 * @param timeMs a 0 or positive value for the new playback position, expressed in ms.
 * @return true if the command to set the playback position was successfully sent.
 * @throws IllegalArgumentException
 */

@@ -6,15 +6,15 @@
     if (timeMs < 0) {
         throw new IllegalArgumentException("illegal negative time value");
     }
-    final int genId;
-    synchronized (mGenLock) {
-        genId = mClientGenerationIdCurrent;
+    synchronized (mInfoLock) {
+        if (mCurrentSession != null) {
+            mCurrentSession.getTransportControls().seekTo(timeMs);
+        }
     }
-    mAudioManager.setRemoteControlClientPlaybackPosition(genId, timeMs);
     return true;
 }

